Description: debianization: defaults
 Meant to take into account /etc/defaults
 in order to avoid security surprises.
 Thanks to Toni Mueller <toni@debian.org>
 for pointing out the issue and providing
 a patch.
Origin: vendor, Debian
Forwarded: not-needed
Author: Toni Mueller <toni@debian.org>
Last-Update: 2020-05-08

--- a/sbin/firehol
+++ b/sbin/firehol
@@ -24,6 +24,15 @@
 #
 #       See the file COPYING for details.
 #
+if [ -r /etc/default/firehol ]; then
+	source /etc/default/firehol
+	if [ ${START_FIREHOL:-NO} != YES ]; then
+		1>&2 echo "FireHOL needs to be enabled in /etc/default/firehol in order to run."
+		1>&2 echo "Not starting - your system may be unprotected!"
+		exit 1
+	fi
+fi
+#
 READLINK_CMD=${READLINK_CMD:-readlink}
 BASENAME_CMD=${BASENAME_CMD:-basename}
 DIRNAME_CMD=${DIRNAME_CMD:-dirname}
